website/docs/configuration/flags.md

```diff
@@ -64,6 +64,18 @@ Some default values are overridden in [our Docker image](quickstart-guide/docker
 | -------------- | ------------------------------------------- | --------------------- | ------------- |
 | `--sqlite-url` | SQLite URI (directory) for 'sqlite' handler | `FERRETDB_SQLITE_URL` | `file:data/`  |
 
+## Query pushdown
```

Comment:
There is a comment on line 16 about the ordering. There is also a comment on line 88 that should be updated.

Reply:
Thanks, reordered and also updated the comment.